storage/google/cloud/storage/client.py

```diff
@@ -309,6 +311,31 @@ def create_bucket(self, bucket_or_name, requester_pays=None, project=None):
         bucket.create(client=self, project=project)
         return bucket
 
+    def download_blob_to_file(self, blob_or_uri, file_obj, start=None, end=None):
+        """Download the contents of a blob object or blob URI into a file-like object.
+
+        Args:
+            blob_or_uri (Union[ \
+                :class:`~google.cloud.storage.blob.Blob`, \
+                 str, \
+            ]):
+                The blob resource to pass or URI to download.
```

Comment:
Let's give an example of the URI `gs://bucket_name/path/to/blob`

Reply:
Added examples for URI and if passing in a resource object.